unitycatalog-ai/integrations/crewai/README.md

```diff
@@ -0,0 +1,211 @@
+# Using Unity Catalog AI with CrewAI 
```

Comment:
Could you rename everything to unitycatalog-crewai? Similar to Ben's PR here https://github.com/unitycatalog/unitycatalog/pull/643